File: test_archives/py_tests/batch3/test_function9.py
from functions.function9 import function9
import pytest

def test_function9_1():
    assert function9('total = ', 1, 2, 3, [1, 2, 3]) == 'total = 24'

def test_function9_2():
    assert function9('sum = ', 2, 4, 6, [3, 5, 7]) == 'sum = 51'

File: test_archives/cpp_tests/batch3/test_function9.cpp
#include "../catch.hpp"
#include "../functions/function9.cpp"

TEST_CASE("Test function9 1", "[function9]") {
  REQUIRE(function9("total = ", 1, 2, 3, std::vector<int>{1,2,3}) == "total = 24");
}

TEST_CASE("Test function9 2", "[function9]") {
  REQUIRE(function9("sum = ", 2, 4, 6, std::vector<int>{3,5,7}) == "sum = 51");
}